src/models/stores/ui.ts

```diff
@@ -95,6 +96,15 @@ export const UIModel = types
         const activeTabState = self.tabs.get(self.activeNavTab);
         return self.openSubTab && activeTabState?.openDocuments.get(self.openSubTab);
       }
+    },
+    get focusSecondaryDocument () {
+      if (self.activeNavTab === ENavTab.kProblems || self.activeNavTab === ENavTab.kTeacherGuide) {
+        const facet = self.activeNavTab === ENavTab.kTeacherGuide ? ENavTab.kTeacherGuide : undefined;
+        return buildSectionPath(self.problemPath, self.openSubTab, facet);
```

Comment:
I think it would be more clear if you broke this into two cases like so:
```suggestion
      if (self.activeNavTab === ENavTab.kProblems) {
        return buildSectionPath(self.problemPath, self.openSubTab);
      } else if (self.activeNavTab === ENavTab.kTeacherGuide) {
        return buildSectionPath(self.problemPath, self.openSubTab, ENavTab.kTeacherGuide);
```

Reply:
I kept the original code because it is essentially a duplicate of the one for the `focusDocument` and wanted to keep the same style.